packages/headless/src/features/facets/facet-set/interfaces/options.ts

```diff
@@ -1,6 +1,8 @@
-import {FacetRequest} from './request';
+import {FacetOptions} from '../../../../product-listing.index';
```

Comment:
Should you not import FacetOptions from the normal controllers instead of product-listing ?

like this import {FacetOptions} from '../../../../controllers';

Reply:
Oh yes, that would make more sense 🙂